code/clue.js

```diff
@@ -1,59 +1,212 @@
+  twoHanded: false,
+  color: 'blue'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const poison = {
+  name: 'Poison',
+  weight: 1,
+  twoHanded: true,
+  color: 'orange'
+}
+
+const axe = {
+  name: 'Axe',
+  weight: 8,
+  twoHanded: true,
+  color: 'red'
+}
+
+const bat = {
+  name: 'Bat',
+  weight: 2,
+  twoHanded: true,
+  color: 'brown'
+}
+
+const trophy = {
+  name: 'Trophy',
+  weight: 4,
+  twoHanded: false,
+  color: 'gold'
+}
+
+const pistol = {
+  name: 'Pistol',
+  weight: 1,
+  twoHanded: false,
+  color: 'purple'
+}
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  profPlum,
+  msScarlet,
+  mrsPeacock,
+  colMustard,
+  mrsWhite,
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
 
-const rooms = []
+const rooms = [
+  "Dining Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+]
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+let mystery = {
+  killer: "",
+  weapon: "",
+  room: ""
+}
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
-
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
   document.getElementById(
     'killerName'
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerImage').src = mystery.killer.image
+  document.getElementById('killerAge').innerHTML = `${mystery.killer.age} years old`
+  document.getElementById('killerOccupation').innerHTML = mystery.killer.occupation
+  document.getElementById('killerDescription').innerHTML = mystery.killer.description
+}
+
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponCard').style.background = mystery.weapon.color
+  document.getElementById('weaponName').innerHTML = mystery.weapon.name
+  if (mystery.weapon.twoHanded === false) {
+    document.getElementById('weaponWeight').innerHTML = `It weighs ${mystery.weapon.weight} kg and is a one-handed weapon.`
+  }
+  else if (mystery.weapon.twoHanded === true) {
```

Comment:
...and this means true!
```suggestion
  else if (mystery.weapon.twoHanded) {
```